prediction/src/algorithms/identify/prediction.py

```diff
@@ -0,0 +1,232 @@
+# limit memory usage..
+import logging
+import os
+
+import numpy
+import pandas
+# limit memory usage..
+from keras import backend as K
+from keras.layers import Input, Convolution3D, MaxPooling3D, Flatten, AveragePooling3D
+from keras.metrics import binary_accuracy, binary_crossentropy, mean_absolute_error
+from keras.models import Model
+from keras.optimizers import SGD
+
+from . import helpers
+
+CUBE_SIZE = 32
+MEAN_PIXEL_VALUE = 41
+EXTRACTED_IMAGE_DIR = "data/extracted/"
+NODULE_DETECTION_DIR = "data/detections/"
+K.set_image_dim_ordering("tf")
+POS_WEIGHT = 2
+NEGS_PER_POS = 20
+P_TH = 0.6
+LEARN_RATE = 0.001
+PREDICT_STEP = 12
+
+
+def prepare_image_for_net3D(img):
+    img = img.astype(numpy.float32)
+    img -= MEAN_PIXEL_VALUE
+    img /= 255.
+    img = img.reshape(1, img.shape[0], img.shape[1], img.shape[2], 1)
+    return img
+
+
+def filter_patient_nodules_predictions(df_nodule_predictions: pandas.DataFrame, patient_id, view_size):
```

Comment:
is that first `:` a typo? :)

Reply:
hmm, possible this is a typehint? I must admit, I haven't used those yet... heh.

Reply:
Yes, it's a typehint. The author also used it e.g. in `def get_net(input_shape=(CUBE_SIZE, CUBE_SIZE, CUBE_SIZE, 1), load_weight_path=None) -> Model:`.
Since that has never been used in our project, I'll remove it again for consistency reasons?

Reply:
Actually, that `->` is what triggered me recognizing it. I don't think you need to remove them per se, but IMHO type hints would be better suited "in addition" to doc blocks as opposed to "in stead of".